.gitlab-ci-check-license.yml

```diff
@@ -79,7 +79,8 @@ test:check-license:golang:
     - go install github.com/google/go-licenses@latest
     - apk add --no-cache git
   script:
-    - go-licenses report . --ignore github.com/mendersoftware > licenses.csv
+    - echo "{{ range . }}{{printf "%s,%s\n" .Name .LicenseName}}{{ end }}" >> format.tpl
```

Comment:
I suggest we check in `format.tpl` in this repo instead of generating in on the fly.

This way a developer can copy it to a repository to manually generate the licenses report (similarly to [code style rules](https://github.com/mendersoftware/mendertesting/blob/master/.clang-format) or [linting rules](https://github.com/mendersoftware/mendertesting/blob/master/.golangci.yml))

Reply:
yeah there I wasn't sure what direction to take, the checked in file is the proper way, but for me the copying from the pipeline and pasting it locally seemed more accessible - but it should now be checked in 👌